src/common/Collision/Management/IVMapManager.h

```diff
@@ -49,6 +49,23 @@ namespace VMAP
     #define VMAP_INVALID_HEIGHT       -100000.0f            // for check
     #define VMAP_INVALID_HEIGHT_VALUE -200000.0f            // real assigned value in unknown height case
 
+    struct TC_COMMON_API AreaAndLiquidData
```

Comment:
*_API macro is only needed when your struct/class has methods implemented in .cpp file